### ISSUE TITLE
Allow For user defined default color

### ISSUE THREAD

**Issue by kevancress** [kevancress/MeasureIt-ARCH#113]
Can the default colour of the annotation be white, or something which complies with WCAG contrast requirements? https://contrastchecker.com/

Right now, black on the Blender default grey is a bit hard to read :)

_Originally posted by @Moult in https://github.com/kevancress/MeasureIt-ARCH/issues/4#issuecomment-664115536_
